### ISSUE TITLE
Projects search

### ISSUE THREAD

**Issue by cfidler** [BNHM/ecoreader#55]
When we catalog GRP materials, we include a GRP topical subject heading (650). This is separates it from the LCSH Geographic (651) field. More and more, we are grouping field notes through expedition topical terms. It would be helpful if researchers could search the EcoReader by the 650 topical term field. Parking this thought here for future enhancements.

**Comment by ccicero** [comment]
Agreed. These project names should match the ones in Arctos. Are you using Arctos project names?

**Comment by cfidler** [comment]
I'm not currently. Of course, the ones I am working on are the Coast surveys which I don't see in Arctos but we can reconcile these.
